internal/policies/mount/adsys_mount/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "adsys_mount"
+description = "Binary for handling the user mounts policy of adsys.\nThis is not meant to be executed manually."
+version = "0.1.0"
+edition = "2021"
+include = ["src/**/*"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "lib"
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = "1.0.65"
+chrono = "0.4.22"
```

`chrono` is unmaintained. I recommend switching to [`time`](https://lib.rs/crates/time) crate:

```suggestion
time = { version = "0.3", features = ["formatting", "macros"] }
```